tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
@geerlingguy I opened up a PR against @exploide branch that would add support for Ubuntu 16.04 out of the box. What do you think? https://github.com/exploide/ansible-role-certbot/pull/3